src/HUDL/HUDL.cpp

```diff
@@ -38,4 +37,24 @@ void HUDL::initLCD() {
     lcd.initLCD();
 }
 
+CO_OBJ_T* HUDL::getObjectDictionary() {
+    return &objectDictionary[0];
+}
+
+uint16_t HUDL::getObjectDictionarySize() const {
+    return OBJECT_DICTIONARY_SIZE;
+}
+
+uint32_t* HUDL::getThermTemps() {
```

Comment:
instead of exposing the thermistor and voltage values here, make a void function called something like `updateLCD()` that will later be used to regularly update the LCD. For now, just put logging statements inside updateLCD to print all the data you want to be able to see.